bundles/config/org.eclipse.smarthome.config.core/src/main/java/org/eclipse/smarthome/config/core/ConfigDescriptionParameterGroup.java

```diff
@@ -0,0 +1,105 @@
+/**
+ * Copyright (c) 2014-2015 openHAB UG (haftungsbeschraenkt) and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.eclipse.smarthome.config.core;
+
+/**
+ * The {@link ConfigDescriptionParameterGroup} specifies information about parameter groups.
+ * A parameter group is used to group a number of parameters together so they can
+ * be displayed together in the UI (eg in a single tab).
+ * <p>
+ * A {@link ConfigDescriptionParameter} instance must also contain the groupId. It should be permissible to use the
+ * groupId in the {@link ConfigDesctiptionParameter} without supplying a corresponding
+ * {@link ConfigDescriptionParameterGroup} - in this way the UI can group the parameters together, but doesn't have the
+ * group information.
+ *
+ * @author Chris Jackson - Initial Contribution
+ *
+ */
+public class ConfigDescriptionParameterGroup {
+
+    private String name;
+    private String context;

```

Comment:
Do we really need a context for groups? the contexts we want define for parameters are terms like "email", "password" and so on. I don´t think that they fit for groups.